File: beecrowd_1160.cpp
#include <stdio.h>

int main() {

  int t, x, y, count = 0;
  double a, b;

  scanf("%d", &t);

  for (; t > 0; --t) {

    scanf("%d %d %lf %lf", &x, &y, &a, &b);

    while (x <= y && count < 101) {

      x += (int)((x * a) / 100);
      y += (int)((y * b) / 100);
      ++count;
    }

    if (count > 100)
      printf("Mais de 1 seculo.\n");
    else
      printf("%d anos.\n", count);

    count = 0;
  }

  return 0;
}

File: Python/beecrowd_1160.py
n = int(input())
for i in range(n):
    pa, pb, g1, g2 = input().split()
    pa = int(pa)
    pb = int(pb)
    g1 = float(g1)
    g2 = float(g2)
    a = 0
    while (pa <= pb and a <= 100):
        cpa = (pa * g1) // 100
        cpb = (pb * g2) // 100
        pa += cpa
        pb += cpb
        a += 1
    
    print("Mais de 1 seculo." if (a > 100) else ("%d anos." % a))
